src/Kdyby/Doctrine/Entities/BaseEntity.php

```diff
@@ -25,6 +25,7 @@
 /**
  * @author Filip Procházka <filip@prochazka.su>
  *
+ * @deprecated
  * @ORM\MappedSuperclass()
  */
 abstract class BaseEntity extends Nette\Object implements \Serializable
```

Comment:
I can see why it could look a bit chaotic :)

The compatibility is broken anyway, so I guess nothing prevents me now from at least making array notation consistent in this project.